* 2009 Nov 22       Kevin Quick        <quick@sparq.org>

    Since /tmp/ is a link under GoboLinux, the mktemp provides more accessible temp files.

--- hg-src/test/run	2009-11-20 22:34:28.000000000 -0700
+++ /Files/Compile/Sources/Golang-hg/test/run	2009-11-22 14:10:08.000000000 -0700
@@ -35,9 +35,12 @@
 
 PATH=/bin:/usr/bin:/usr/local/bin:${GOBIN:-$HOME/bin}:`pwd`
 
-RUNFILE=/tmp/gorun-$$-$USER
-TMP1FILE=/tmp/gotest1-$$-$USER
-TMP2FILE=/tmp/gotest2-$$-$USER
+#RUNFILE=/tmp/gorun-$$-$USER
+#TMP1FILE=/tmp/gotest1-$$-$USER
+#TMP2FILE=/tmp/gotest2-$$-$USER
+RUNFILE=`mktemp -t`
+TMP1FILE=`mktemp -t`
+TMP2FILE=`mktemp -t`
 
 # don't run the machine out of memory: limit individual processes to 4GB.
 # on thresher, 3GB suffices to run the tests; with 2GB, peano fails.
